### ISSUE TITLE
how to use this to do object detectoin

### ISSUE THREAD

**Issue by Sherlooock** [hankyul2/maxvit-pytorch#2]

**Comment by hankyul2** [comment]
Hi @Sherlooock 

You can easily apply this model to Cascade Mask R-CNN using the [mmdetection](https://github.com/open-mmlab/mmdetection) framework. If you want to know the details of the hyper-parameter, refer to the below description from the paper.
![image](https://github.com/hankyul2/maxvit-pytorch/assets/31476895/750c2ffe-f17a-4530-a27d-11793a52f428)
![image](https://github.com/hankyul2/maxvit-pytorch/assets/31476895/744abbb4-be07-4ce2-9bcd-84e53251a15d)


Thank you.

Hankyul
